docs/hub/models-adding-libraries.md

```diff
@@ -180,4 +182,8 @@ All third-party libraries are Dockerized, so you can install the dependencies yo
     pytest -sv --rootdir docker_images/your-awesome-library/docker_images/your-awesome-library/
     ```
 
+### Register your libraries supported tasks on the hub
+
+To register the tasks supported by your library on the hub you'll need to add a mapping from your library name to its supported tasks in this [file](https://github.com/huggingface/hub-docs/blob/main/js/src/lib/interfaces/LibrariesToTasks.ts). This will ensure the inference API is registered for tasks supported by your model. 
```

Comment:
Maybe mention this is automatically generated in the GitHub Actions workflow from the inference PR

Reply:
I'm not sure I understand this. Do you mean that this task mapping is automatically created when a PR is opened for the inference library?

Reply:
Yes, see https://github.com/huggingface/api-inference-community/actions/workflows/python-api-export-tasks.yaml https://github.com/huggingface/api-inference-community/actions/runs/5132896423/jobs/9234786914 as an example

Reply:
Ah, that makes sense. I'll update now.